src/Microsoft.Azure.SignalR.Common/Constants.cs

```diff
@@ -77,6 +77,7 @@ public static class CustomizedPingTimer
         {
             public const string ServiceStatus = "ServiceStatus";
             public const string Servers = "Servers";
+            public const string DiagnosticLogs = "DiagnosticLogs";
```

Comment:
nit.. are these logs for diagnostics only? if not maybe change the name to MessageTracingControl or EnableMessageTracing/Logging?

Reply:
I reuse the concept [Azure Diagnostic Log](https://docs.microsoft.com/en-us/azure/cdn/cdn-azure-diagnostic-logs). I want to keep the same. The message log is part of the azure diagnostic log of ASRS to be onboard. We also have a blade in portal for this log (in Diagnostic Settings blade).